services/frontend-v3/src/components/layouts/appLayout/footer/FooterView.jsx

```diff
@@ -0,0 +1,155 @@
+  /* Component Styles */
+  const styles = {
+    footer: {
+      backgroundColor: "#F8F8F8",
+      paddingLeft: "60px",
+      zIndex: 4,
+      width: "100%",
+
+      left: "0",
+      bottom: "0",
+    },
+    link: {
+      display: "inline",
+      marginLeft: "15px",
+    },
+    aroundLinks: {
+      textAlign: "center",
+      paddingLeft: "270px",
+    },
+    aroundLinksSmall: {
+      textAlign: "center",
+    },
+  };
+
+  // update width to screen size
+
+  const [windowWidth, setWindowWidth] = useState(window.innerWidth);
+
+  const updateWidth = () => setWindowWidth(window.innerWidth);
+
+  useEffect(() => {
+    window.addEventListener("resize", updateWidth);
+
+    return () => window.removeEventListener("resize", updateWidth);
+  }, []);
+
+  if (windowWidth < 993) {
+    return (
+      <Footer style={styles.footer}>
+        <div style={styles.aroundLinksSmall}>
+          <a
+            style={styles.link}
+            // href to be changed to route w about page
+            href="/about"
+          >
+            <FormattedMessage id="footer.about.link" />
+          </a>
+
+          <a
+            style={styles.link}
+            // href to be changed to route w Contact Us page
+            href="/about/help"
+          >
+            <FormattedMessage id="footer.contact.link" />
+          </a>
+
+          <a style={styles.link} href="//about/terms">
+            <FormattedMessage id="footer.terms.and.conditions.link" />
+          </a>
+
+          <a style={styles.link} href="/about/privacy">
+            <FormattedMessage id="footer.privacy.link" />
+          </a>
+        </div>
+      </Footer>
+    );
+  }
+
+  // check ending of url and if url is one of the cases below, apply styling for
+  // footer on pages without sidebar
+
+  const urlSections = window.location.toString().split("/");
+  const endURL = urlSections[urlSections.length - 1];
+
+  if (
+    endURL === "home" ||
+    endURL === "about" ||
+    endURL === "terms" ||
+    endURL === "privacy" ||
+    endURL === "help" ||
+    endURL === ""
+  ) {
+    return (
+      <Footer style={styles.footer}>
+        <div style={styles.aroundLinksSmall}>
+          <a
+            style={styles.link}
+            // href to be changed to route w about page
+            href="/about"
+          >
+            <FormattedMessage id="footer.about.link" />
+          </a>
+
+          <a
+            style={styles.link}
+            // href to be changed to route w Contact Us page
+            href="/about/help"
+          >
+            <FormattedMessage id="footer.contact.link" />
+          </a>
+
+          <a style={styles.link} href="/about/terms">
+            <FormattedMessage id="footer.terms.and.conditions.link" />
+          </a>
+
+          <a style={styles.link} href="/about/privacy">
+            <FormattedMessage id="footer.privacy.link" />
+          </a>
+        </div>
+      </Footer>
+    );
+  }
+
+  // default footer style with sidebar
```

Comment:
I would delete all of this, doing this means there's something we're not doing right in our layouts